debug/programs/trigger.S

```diff
@@ -40,14 +40,14 @@ write_loop:
         j       main_exit
 
 write_store_trigger:
-        li      a0, (1<<6) | (1<<1)
+        li      a0, (2<<60) | (2<<28) | (1<<6) | (1<<1)
```

Comment:
```suggestion
        /* 2<<60 is for RV64. 2<<28 is for RV32. That's safe because on RV64 bits 28 and 29 are 0. */
        li      a0, (2<<60) | (2<<28) | (1<<6) | (1<<1)
```